manifests/init.pp

```diff
@@ -154,8 +155,13 @@
   create_resources('dhcp::pool', $pools)
   create_resources('dhcp::host', $hosts)
 
+  $_service_ensure_real = $service_ensure ? {
+    'unmanaged' => undef,
+    default     => $service_ensure,
+  }
+  
   service { $servicename:
-    ensure => running,
+    ensure => $_service_ensure_real,
     enable => true,
```

Comment:
You should determine this as well. Having `ensure => stopped` but `enable => true` is inconsistent.